src/pages/Game.tsx

```diff
@@ -12,9 +17,11 @@ import GameComponent from '../components/GameComponent';
 import GameChat from '../components/GameChat';
 import GameWrapper from '../styles/GameStyle';
 import { Message } from '../types/AppTypes';
+import UserContext from '../context/Context';
 
 function Game() {
   const navigate = useNavigate();
+  const { user }: any = useContext(UserContext);
```

Comment:
Try getting rid of "any" and making sure you're getting the right user object type with all the required props.